pubspec.yaml

```diff
@@ -29,6 +29,9 @@ dependencies:
   # file
   path_provider: ^2.0.11
 
+  #UI
+  stack: ^0.0.1
```

Comment:
any insight what's this for? 🤔

Reply:
this was the one I used initially for the UI but boss @Jotam suggest to use rows and columns as this is prone to overlaps

Reply:
so will it still be used anyways? If not feel free to remove 🙂

Reply:
yes it will not be used anymore